waspc/data/Generator/templates/server/rollup.config.js

```diff
@@ -0,0 +1,37 @@
+import esbuild from 'rollup-plugin-esbuild'
+
+/** @type {import('rollup')['RollupOptions']} */
+export default [
+  {
+    input: 'src/server.ts',
+    plugins: [
+      esbuild({
+        target: 'esnext',
+      }),
+    ],
+    output: [
+      {
+        file: `bundle/server.js`,
+        format: 'es',
+        sourcemap: true,
+      },
+    ],
+    external: (id) => !/^[./]/.test(id),
```

Comment:
I saw this pattern in the wild, it basically means: anything that's not a relative import, it's a dependency and don't try to bundle it together. https://rollupjs.org/configuration-options/#external

Reply:
Interesting! We should certainly provide this comment inside this file, since it won't be obvious.